cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -61,14 +61,14 @@ Replacement syntax for deprecated and removed features are also indicated.
 | `ON HOME GRAPH` for privilege commands        | Syntax        | Added         | New syntax for privileges affecting home graph
 | `ON DEFAULT DATABASE` for privilege commands  | Syntax        | Deprecated    | Replaced by `ON HOME DATABASE`
 | `ON DEFAULT GRAPH` for privilege commands     | Syntax        | Deprecated    | Replaced by `ON HOME GRAPH`
-| `CREATE [BTREE] INDEX [name] FOR ()-[r:TYPE]-() ON (r.prop) [OPTIONS {...}]`  | Functionality | Added | Allows creating indexes on relationships with a particular relationship type and property combination. They can be dropped by using their name.
-| `CREATE LOOKUP INDEX [name] FOR (n) ON EACH labels(n) [OPTIONS {}]`           | Functionality | Added | Create lookup index for nodes with any labels. They can be dropped by using their name.
-| `CREATE LOOKUP INDEX [name] FOR ()-[r]-() ON [EACH] type(r) [OPTIONS {}]`     | Functionality | Added | Create lookup index for relationships with any types. They can be dropped by using their name.
+| `CREATE INDEX FOR ()-[r:TYPE]-() ...`         | Functionality | Added         | Allows creating indexes on relationships with a particular relationship type and property combination. They can be dropped by using their name.
```

Comment:
```
| `+CREATE INDEX <index_name> FOR ()-[r:TYPE]-() ...+`
| Functionality
| Added
a| Create an _Index_ on relationships with a particular relationship type and property combination.
An index can be dropped by using the name of the index.
```

Reply:
we don't have this on any of the lines in this file so it feels odd to add them to just these ones. Feels like that should be a separate clean-up to fix for everything if it is wanted

Reply:
Sure, I can do a cleanup later on.